### PR TITLE
q.10310 cleanup

### PR DESCRIPTION
**Pull request by killerwife**
Added missing visuals, fixed timers and counts. AI will no longer attack
back.

### CODE REVIEW

**Comment by killerwife** [comment]
('42456','0','0','9','76','47','0','0','185318','0','0','1','Summon smoke object'),
('42467','0','0','8','76','47','0','0','185319','0','0','1','Summon fire object'),

Need to be added to wotlk if this is merged.

File: src/scriptdev2/scripts/outland/netherstorm.cpp [hunk 1]
```diff
@@ -1194,22 +1195,29 @@ struct npc_drijyaAI : public npc_escortAI
...
             m_uiSpawnDestroyerTimer = 0;
             m_uiDestroyingTimer     = 0;
+            m_uiSayCount            = false;
         }
     }
 
+    void MoveInLineOfSight(Unit* pWho) override
+    {
+        if (pWho->GetTypeId() != TYPEID_PLAYER)
+            return;
```
Comment (boxa) [inline src/scriptdev2/scripts/outland/netherstorm.cpp]: forgotten?
            npc_escortAI::MoveInLineOfSight(pWho);

Comment (killerwife) [inline src/scriptdev2/scripts/outland/netherstorm.cpp]: Nope, creature should not attack back.

Comment (boxa) [inline src/scriptdev2/scripts/outland/netherstorm.cpp]: hm. why not empty
void MoveInLineOfSight(Unit* pWho) override { }

Comment (killerwife) [inline src/scriptdev2/scripts/outland/netherstorm.cpp]: Thats a good question indeed :D

**Pull request merged**